### PR TITLE
[install CentOS] PDO class not found fix for centos

### PR DESCRIPTION
**Pull request by maltheism**
## Brief summary of changes

Solves PDO class not found error on CentOS.

See https://github.com/aces/Loris/issues/5452

### CODE REVIEW

File: README.CentOS7.md [hunk 1]
```diff
@@ -36,6 +36,11 @@ sudo yum update
 sudo yum install php72
 sudo yum install php72-php-fpm php72-php-gd php72-php-json php72-php-mbstring php72-php-mysqlnd php72-php-xml php72-php-xmlrpc php72-php-opcache php72-php-pdo php72-php-mysql
 ```
+### PHP MySQL Packages
```
Comment (PapillonMcGill) [inline README.CentOS7.md]: same as for #5485, not the right fix.
#5489 will fix all 3 issues

Comment (maltheism) [reply README.CentOS7.md]: Hi @PapillonMcGill, I was getting these output errors after #5489. I think this PR and #5485 are actually required.

Comment (maltheism) [reply README.CentOS7.md]: @PapillonMcGill I'll try re-testing on a fresh install of CentOS and see what happens.

Comment (maltheism) [reply README.CentOS7.md]: @PapillonMcGill this PR is needed to resolve the PDO not found https://github.com/aces/Loris/pull/5533#discussion_r344882512

**Comment by PapillonMcGill** [comment]
I could approve but it will conflict with #5485.

**Comment by maltheism** [comment]
@driusan I fixed the rebase. This PR would be nice to get merged for release because it solves an installation problem on centOS.

File: README.CentOS7.md [hunk 2]
```diff
@@ -32,8 +32,8 @@ sudo yum install epel-release
 sudo yum install http://rpms.remirepo.net/enterprise/remi-release-7.rpm
 sudo yum install yum-utils
 sudo yum update
-sudo yum --enablerepo=remi-php72 install php php-xml php-mbstring
-sudo yum install php72-php-fpm php72-php-gd php72-php-json php72-php-mysqlnd php72-php-xmlrpc php72-php-opcache php72-php-pdo php72-php-mysql
+sudo yum --enablerepo=remi-php72 install php php-pdo php-pdo_mysql php-xml php-mbstring
+sudo yum install php72-php-fpm php72-php-gd php72-php-json php72-php-mbstring php72-php-mysqlnd php72-php-xml php72-php-xmlrpc php72-php-opcache php72-php-mysql
```
Comment (PapillonMcGill) [inline README.CentOS7.md]:
```suggestion
sudo yum install php72-php-fpm php72-php-gd php72-php-json php72-php-mysqlnd php72-php-xmlrpc php72-php-opcache php72-php-mysql
```
packages installed on the previous line

Comment (maltheism) [reply README.CentOS7.md]: @PapillonMcGill committed your suggestion. Should I change it as your alternative suggestion?

Comment (PapillonMcGill) [reply README.CentOS7.md]: I believe so. It would be only 1 instruction and it would be more consistent.

Comment (maltheism) [reply README.CentOS7.md]: @PapillonMcGill I added it.

**Pull request merged**